modules/openapi-generator/src/main/resources/go/api_doc.mustache

```diff
@@ -35,6 +35,11 @@ import (
     {{goImportAlias}} "./openapi"
 )
 
+type openapiError interface {
+    Model() interface{}
+    Body() []byte
+}
+
```

Comment:
Not sure I understand the need for a custom interface. You can directly do `err.(GenericOpenAPIError)`

Reply:
Yup. Will do that.